code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -1870,12 +1848,12 @@ func (srh *StorageRestHandler) getReadMarkers(w http.ResponseWriter, r *http.Req
 		return
 	}
 
-	rms := make([]readMarkerResponse, len(readMarkers))
+	rmrs := make([]ReadMarkerResponse, len(readMarkers))
```

Comment:
Should be `rmrs := make([]ReadMarkerResponse, 0, len(readMarkers))` if we use append to add items below.